modules/core/src/main/scala/eu/timepit/fs2cron/package.scala

```diff
@@ -29,17 +29,20 @@ import fs2.Stream
 import scala.concurrent.duration.FiniteDuration
 
 package object fs2cron {
+  val defaultZone: ZoneId = ZoneId.systemDefault()
```

Comment:
This looks like a side-effect that is not suspended in `F`. I would prefer to not use a default argument and always let the user pass a `ZoneId` explicitly.

Reply:
Or the `zoneId` parameter could be made optional (`Option[ZoneId]`) and `ZoneId.systemDefault()` is then used in `durationFromNow` (or `evalNow`) where it could properly suspended in `F` if the parameter is `None`.